src/Components/Wizard/types.tsx

```diff
@@ -0,0 +1,23 @@
+import { FormikHandler } from "../Forms/support"
+
+export interface RenderProps {
+  validate?: () => any | FormikHandler
+  validationSchema?: () => any | FormikHandler
+  next: (v: any) => void
+  previous: (v: any) => void
+  onComplete: (v: any) => void
+  pageIndex: number
+  pages?: WizardSchema
```

Comment:
I'd name this variable `steps: WizardSteps`. Just to be aligned with the `Step` component